CHANGELOG.asciidoc

```diff
@@ -236,6 +236,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Add Elasticsearch index_summary metricset. {pull}6918[6918]
 - Add config option `management_path_prefix` for RabbitMQ module to configure management plugin path prefix {issue}6875[6875] {pull}7074[7074]
 - Add shard metricset to Elasticsearch module. {pull}7006[7006]
+- Set guest as default user in RabbitMQ module. {pull}7107[7107]
```

Comment:
Sounds a bit like a breaking change. Or was it before a bug?

Reply:
I think it is an enhancement for the out-of-the-box case, it can be breaking if someone has managed to setup the RabbitMQ endpoint without authentication, a very corner case if even possible.

If there are other authentication mechanisms configured in RabbitMQ, probably plain authentication headers are ignored, but not sure. I can try to test if this breaks this case.